src/sagemaker/jumpstart/types.py

```diff
@@ -2116,6 +2117,7 @@ class JumpStartModelInitKwargs(JumpStartKwargs):
         "hub_content_type",
         "model_reference_arn",
         "specs",
+        "accept_draft_model_eula",
```

Comment:
Can we instead move this field to the `JumpStartModelDeployKwargs` like [this one](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/jumpstart/types.py#L2234)?

Also, I do not think this field is extensible to multiple eula acceptance for new use cases

Reply:
ack makes sense. this way our display_benchmark_metrics method is not impacted by validations at initialization time